reporting/src/request/steps/token-telemetry/cached-entry-pipeline.js

```diff
@@ -19,16 +17,19 @@ const CACHE_TTL = 2 * 24 * 60 * 60 * 1000; // 2 days
  * Abstract part of token/key processing logic.
  */
 export default class CachedEntryPipeline {
-  constructor({ name, db, trustedClock, primaryKey, options }) {
-    this.name = name;
-    this.db = db;
-    this.trustedClock = trustedClock;
+  constructor(options) {
```

Comment:
Perhaps better to call it something like "params", because it is easy to confused "options.options" and "options".